github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```
@@ -387,3 +389,102 @@ periodics:
       - name: gcs
         secret:
           secretName: gcs
+- name: periodic-kubevirt-e2e-k8s-latest
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+  cron: "0 6,14,22 * * *"
+  decorate: true
+  decoration_config:
+    timeout: 7h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: master
+    work_dir: true
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - "export TARGET=k8s-1.20 && automation/test.sh"
```

nit: make this a little more agnostic, i.e: 
```suggestion
          - "export TARGET=$(ls cluster-up/cluster/ | grep -E '^k8s-1\.[0-9]+$' | sort --sort=version -r | sed -n '1p') && automation/test.sh"
```

---

> nit: make this a little more agnostic, i.e:

awesome, much better! :heart_eyes: